include/SFML/Window/Keyboard.hpp

```diff
@@ -153,9 +153,16 @@ enum Key
     F15,          //!< The F15 key
     Pause,        //!< The Pause key
 
-    KeyCount, //!< Keep last -- the total number of keyboard keys
+    Last = Pause, //!< Keep equal to last enumerator -- used to define KeyCount
 };
 
+////////////////////////////////////////////////////////////
+/// \brief The total number of keyboard keys, ignoring Key::Unknown
+///
+////////////////////////////////////////////////////////////
+// NOLINTNEXTLINE(readability-identifier-naming)
+static constexpr unsigned int KeyCount{static_cast<unsigned int>(Key::Last) + 1};
```

Comment:
Is this correct? Because it certainly changes the meaning to the past. Before `KeyCount == Key::Last`, but now `KeyCount == Key::Last + 1` which is not the same.

Reply:
Before we had `KeyCount == Key::Pause + 1`,
After we have `KeyCount == Key::Last + 1` but `Key::Last` is `Key::Pause` so `KeyCount == Key::Pause + 1`.

Reply:
Ah, I get it now